Makefile

```diff
@@ -351,4 +351,16 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS)
+	if [ -d /tmp/assisted_test_infra_logs ]; then rm -rf /tmp/assisted_test_infra_logs; fi
```

Comment:
No, see below, needed to remove the session scope.
How can you tell whats run inside the container and what not?

Reply:
Ho i understand now. But why does it matter? tmp is mounted

Reply:
It makes sense if all of our logical workflows would run inside of a container. Makefile shouldn't hold code logic, it's just a nice way to wrap everything up.
In addition, I'd like to avoid code duplication.

Reply:
this logic can be moved to its own make target, or a shell script that gets called from a make target wdyt?

Reply:
First of all lets delete all the "if" logic from here. Just rm -rf and mkdir -p enough

Reply:
tried with separated target, didn't work, ill try again

Reply:
Fixed